packages/slice-machine/components/ChangesEmptyPage/NoChangesPage.tsx

```diff
@@ -1,33 +1,56 @@
-import React from "react";
-import { MdDone } from "react-icons/md";
-import { Flex, Text } from "theme-ui";
+import router from "next/router";
+
+import {
+  BlankSlate,
+  BlankSlateImage,
+  BlankSlateTitle,
+  BlankSlateDescription,
+  BlankSlateActions,
+  BlankSlateContent,
+} from "@src/components/BlankSlate";
+
+import { Button, Image, tokens } from "@prismicio/editor-ui";
+
+import { FiExternalLink } from "react-icons/fi";
+import { TextLink } from "@src/components/TextLink";
+import { useSelector } from "react-redux";
+import { SliceMachineStoreType } from "@src/redux/type";
+import { getDocumentsListEndpoint } from "@src/modules/environment";
 
 export const NoChangesPage = () => {
+  const { documentsListEndpoint } = useSelector(
```

Comment:
I think we need to continue our discussion for the "redux removal" task. Here it's so easy to use a new selector compare to thinking of a new way. 
But I'm afraid that the goal to remove redux is not a good one if we don't try, step by step, to do it.

At the end for your use case it's super simple what you did

Reply:
In this case it's a bit hard if there is not a top-level query of environment values. I think moving selectors would be very easy once it's done. WDYT?

Reply:
Yes, I agree that here we don't have top level env values. Maybe that's a case where we would need Context? :D